### ISSUE TITLE
Video Game | Gaming | Entertainment Addiction

### ISSUE THREAD

**Issue by nelsonic** [nelsonic/nelsonic.github.io#779]
This is a post about the perils of entertainment and _distraction_ at the expense of _traction_ toward a specific and meaningful goal. It's _personal reflection_ and not meant criticism of anyone `else`.

Yesterday I spent **6 hours** (17:00 - 23:00) playing [Horizon Zero Dawn](https://en.wikipedia.org/wiki/Horizon_Zero_Dawn) (HZD) on PS4. 🎮  
If you're into games and haven't played it, see the [Gameplay](https://youtu.be/T5Xx3MdqdgM) it's _incredible_. 
Of the 2439 games available for PS4, HZD is the [_fifth_ most popular](https://en.wikipedia.org/wiki/List_of_best-selling_PlayStation_4_video_games) with good reason.
The graphics put CGI of most big budget movies from the 2000's to shame
and the level of emersion is like nothing I've ever played. 

While I _enjoyed_ the time I spent playing the game, I felt it was "_wasted_" time because I could have invested the time into building _features_ for our [App](https://github.com/dwyl/app)! I _thoroughly disagree_ with the following quote: 

###  “_Time you enjoy wasting is not wasted time_.” ~ [Marthe Troly-Curtin](https://www.goodreads.com/quotes/29449-time-you-enjoy-wasting-is-not-wasted-time)

I will _not_ be playing any more of HZD regardless of how "fun" it is/was. 
Experienced gamers take around [60 - 80 hours](https://www.fraghero.com/heres-how-long-it-actually-takes-to-beat-horizon-zero-dawn) to complete the game 
and I simply don't have that kind of time to "spare". 

I don't feel that I am "addicted" to playing games. I don't have _any_ games on my Computer or Phone (_the devices I use most_) and can go for _weeks_ without thinking about or touching the Nintendo Switch of PS4. But playing for 6h yesterday (_and I would have continued if I didn't need to be at my desk for work this morning!_) gave me a _glimpse_ into how _addictive_ highly immersive games can be!
I'm very glad I've resisted the temptation to play online multi-player games like League of Legends (LoL) or Fortnite which by all accounts are the virtual equivalent of crack and meth. 

<hr />

# tl;dr > decision required!

I think we need to make a _decision_ about Video Games [**`@home`**](https://github.com/dwyl/home).
We _either_ need to ***outright ban*** all forms of video games (_phone, PC and console!_)
***or*** have a _strict_ rule that all games must be _social_ with other humans in the room. 
i.e. **No open world _single_ player games** where the player plays by themself for _weeks_
or online games where you play with strangers or (virtual) "friends" around the world. 

The purpose of [**`@home`**](https://github.com/dwyl/home) is to be a place where people can learn and _create_. 
While games can be creative for the _maker_, they aren't for the _player_. 

<hr />

# Gaming _Addiction_

The World Health Organisation (WHO) has included "**gaming disorder**" in the 11th revision of the International Statistical Classification of Diseases and Related Health Problems (ICD-11) released in June 2018. https://en.wikipedia.org/wiki/Video_game_addiction#World_Health_Organization

In` 2019, the world's first screening tool for Gaming Disorder as defined by the WHO was published in a journal article in the [International Journal of Mental Health and Addiction](https://link.springer.com/journal/11469). The Gaming Disorder Test (GDT) includes the instruction to answer the following four questions using the following options:

Never | Rarely | Sometimes | Often | Very Often
-- | -- | -- | -- | --

1. I have had **difficulties controlling** my **gaming activity**.
2. I have **given increasing priority** to **gaming over other life interests** and daily activities.
3.  I have **continued gaming** despite the occurrence of **negative consequences**.
4.  I have experienced **significant problems** in life (e.g., personal, family, social, education, occupational) **due to** the severity of my **gaming behaviour**.

## Videos on the Topic

Escaping video game addiction: Cam Adair
[![image](https://user-images.githubusercontent.com/194400/73163988-eeba5280-40e8-11ea-9e70-8bb68f4158b3.png)](https://youtu.be/EHmC2D0_Hdg)

Fast forward to [02:40](https://youtu.be/EHmC2D0_Hdg?t=149) ...
> "_Unfortunately, the debate surrounding video games focusses on whether you should play or not.
> When that's like saying should you drink or not. If you can do it in moderation, that's fine. 
> But what if you can't. What if right now you're stuck at home playing video games and you **want to stop** and don't know how. Imagine for a second how this makes you feel. 
> Do you feel a sense of **pain**? What about feelings of **guilt**? ..._"

> [03:10](https://youtu.be/EHmC2D0_Hdg?t=184) "_It's not about the games, it's about **why** you play games.
> If you understand **why** you play games, you can move on from them._"

> "_There are four main reasons why you play games_:
> 1. **Temporary _Escape_** - the perfect way of not having to deal with situations in your [real] life. Simply get absorbed in games and play for hours and hours.
> 2. **They are _Social_** - Playing with your friends online. Games offer a _clean slate_ on the social ladder. Being bullied [IRL] does not leave people feeling confident in their social standing. Feeling unaccepted [IRL] leads people to seek new social networks online.
> 3. **The are a _Challenge_** - They give you a sense of purpose, a mission, a goal to work towards. This is an _achievement_ paradigm. The opportunity to experience success. 
> 4. **Constant measurable _Growth_** - the feedback loop, you get to see [progress](https://github.com/nelsonic/nelsonic.github.io/issues/521#issuecomment-538769171). Online you are able to see _rewards_ for the efforts you put in.

> Combine these four areas and you have a ***very addicting process***.

> So where do we go from here? How do we fix this problem?
> Video game addiction is a habit developed over time by becoming your go-to activity whenever you are bored. 

![image](https://user-images.githubusercontent.com/194400/73170709-07316980-40f7-11ea-879a-81cfe5a3b311.png)

> "_So parents, it starts with you. 
> The iPad is not the new baby sitter. 
> They need interaction _not_ entertainment. 
> Gamers play for very specific reasons. 
> Identify their motivations and help them find these in other activities. 
> Help them with their social skills. 
> The truth is they struggle to make friends.
> Lastly, don't punish them for their desire to play these games.
> Come from a place of compassion and encouragement, not judgement. 
> We're so caught up in asking whether this is a "real" addiction or not that we've lost sight of what truly matters. **_How_ do we _help_ these people _stop_ playing video games**?_"

> "_There is another way. This is about feeling trapped in something you want to move on from ...
You have permission to move on_."

Comments on the video: 
![image](https://user-images.githubusercontent.com/194400/73163335-b108fa00-40e7-11ea-9698-1cceb898bb9e.png)
Seriously, take a minute to read a few of the comments, this video _really_ resonates with a lot of people!

Gaming Disorder: What It’s Like to Be Addicted to Video Games
[![game-addition-lol-advert](https://user-images.githubusercontent.com/194400/73166714-4b6c3c00-40ee-11ea-8a20-c3854265ad4b.png)](https://youtu.be/QUcr5nZcQLU)

Does anyone ***`else`*** see the irony in YT serving an Ad for LoL on a video about gaming addiction?

Top comment:
![image](https://user-images.githubusercontent.com/194400/73171634-1dd8c000-40f9-11ea-9ee3-8285e085ca17.png)
YouTube addiction is a topic for another post, but it's relevant. 

Gaming Addiction: When Does Video Gaming Become a Problem?
[![image](https://user-images.githubusercontent.com/194400/73171806-845dde00-40f9-11ea-829b-4f34efcaf4c5.png)](https://youtu.be/u3avMv4v-j0)

Watch **Rehab for Korea's Gaming Addicts** decide if this is any _less_ addictive than alcohol or drugs. 
[![image](https://user-images.githubusercontent.com/194400/73177156-284d8680-4106-11ea-9067-6abd2734bb5a.png)](https://youtu.be/uOm5aXXjzzM)
https://youtu.be/uOm5aXXjzzM
1 in 10 (10%) of Korean teenagers are addicted to online gaming. 
It might not _appear_ to be as destructive to individuals or society as the substances traditionally associated with addiction, because the addicts aren't stealing or begging to fuel their addiction.  Consider that many have completely excluded themselves from society 
![image](https://user-images.githubusercontent.com/194400/73177436-ef61e180-4106-11ea-9844-2e32d5ecd8af.png)
> "_It's not just **hours online** that determine an addiction. Diagnoses comes from how a person **behaves** when they are online and how they **feel** when they are not._"

[![image](https://user-images.githubusercontent.com/194400/73178692-b2e3b500-4109-11ea-8b34-81db17c93d5b.png)](https://youtu.be/7GVU9IQ-oP0)

Online game addiction is a **_real_ problem**. 
Korea/China/Japan are "ahead" on this in that the trend was reported in 2014.
If 97% of young people aged 10 - 25 are playing games and 10% are becoming addicted to the point that they miss school/work, there is an epidemic of real-life social maladaptation coming.
From a societal perspective, _maybe_ entertainment for the masses of unemployed people is a _good_ thing. But are the games making the world _better_ or simply distracting people away from doing something meaningful?

**Comment by Hilary325** [comment]
With the increasing amount of games, there is no surprise that a problem of gaming addiction has become common. Fortunately, there exist free rehab centers like addictionresource that can solve the problem. I know people who had to receive treatment in such institutions, and all of them started a new life after that. And I'm sure, your information will be helpful for many people.

**Comment by nelsonic** [comment]
https://www.theverge.com/2020/5/6/21249497/fortnite-350-million-registered-players
![image](https://user-images.githubusercontent.com/194400/81285858-0e520700-9058-11ea-8c6c-248397064442.png)
**3.2 _billion_ hours**. 
The average person lives **27,375** days (75 years).
16 waking hours per day.
27,375 x 16 = 438,000 hours
3,200,000,000/438,000 = **7,306 life times _wasted_ per month** playing **Fortnite**. 
This _cannot_ be good for humanity/society.

While this is an absurd number, it _pails_ in comparison to how much time people are wasting across Facebook's platforms ... 🙄 
https://www.statista.com/statistics/346167/facebook-global-dau/
Fb has 1.73 billion daily active users spending an average of 37-minutes per day on their sites/apps.
1,730,000,000 x 37 / 60 = 1.066 Billion Hours per day.

1,730,000,000 x 37 / 60 * 365 / 12 = 32.45 Billion Hours per month. 
74,086 life times per month.
The Facebook machine is draining the lives of **2,470 people per day**.
_Imagine_ what all those people could _do_ if they had a tool to help them focus on their important goals/work instead of clicking their lives away on Fb/Insta/etc?!

**Comment by nelsonic** [comment]
Covid19 has accelerated children's addiction to their mobile devices.

The YouTube App is used by 69% of U.S. kids, 74% of kids in the U.K. and 88% of kids in Spain. 
![image](https://user-images.githubusercontent.com/194400/84004890-2cb36700-a964-11ea-9de3-a6821733376a.png)

Children in the US are spending 5+ hours per day on their phones spread across YouTube, TikTok, Instagram, Snapchat, etc. 

https://techcrunch.com/2020/06/04/kids-now-spend-nearly-as-much-time-watching-tiktok-as-youtube-in-u-s-u-k-and-spain/

**Comment by nelsonic** [comment]
Video game play is positively correlated with well-being:
https://www.oii.ox.ac.uk/news/releases/groundbreaking-new-study-says-time-spent-playing-video-games-can-be-good-for-your-wellbeing
PDF: [video-games-wellbeing.pdf](https://github.com/nelsonic/nelsonic.github.io/files/5554387/video-games-wellbeing.pdf)


Obviously the popular media have picked up this research and run with it:
![image](https://user-images.githubusercontent.com/194400/99408966-be96a700-28e8-11eb-8d02-023378bec039.png)

But it does not _compare_ playing games to other activities like reading, exercise or spending time with loved ones.
I would wager that physical activity, _especially_ doing a sport with measured progress like a martial art, is _considerably_ more beneficial - both short and long-term - than sitting on a sofa playing games. 💭

**Comment by iteles** [comment]
I'd be quite interested in reading this whole study to understand whether they also compare different types of games, external factors and relationships. Thanks for including the PDF 👍

**Comment by nelsonic** [comment]
@iteles note: “_This work has not been peer reviewed and is subject to change._”
